### PR TITLE
Bump to 0.8.4 version

### PR DESCRIPTION
**Pull request by pfreixes**
We need to have [this](https://github.com/djc/bb8/pull/207) merged first

### CODE REVIEW

File: bb8/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bb8"
-version = "0.8.3"
+version = "0.9.0"
```
Comment (djc) [inline bb8/Cargo.toml:3]: Why not 0.8.4? I think all of your changes have been semver-compatible?

Comment (pfreixes) [reply bb8/Cargo.toml:3]: Since its adding a new feature and not patching anything got the feeling that minor bump would be the most suitable one to bump.

Comment (pfreixes) [reply bb8/Cargo.toml:3]: And we keep compatibility

Comment (djc) [reply bb8/Cargo.toml:3]: That's not how semver works in Cargo: 0.x to 0.y is a semver-incompatible bump, there is no distinction between feature and fix versions in 0.x versions.

Comment (pfreixes) [reply bb8/Cargo.toml:3]: I was not aware about this change from Cargo compared to what we have historically used major, minor and patched versions.

Comment (pfreixes) [reply bb8/Cargo.toml:3]: Ill put the 0.8.4

Comment (pfreixes) [reply bb8/Cargo.toml]: Done, and thanks for the insights here about how Cargo works with versions

**Review by djc — APPROVED** [review]

**Comment by djc** [comment]
* Published bb8 v0.8.4 at registry `crates-io`
* [new tag]         v0.8.4 -> v0.8.4

Thanks!

**Comment by pfreixes** [comment]
Seems that the documentation is not providing all the details https://docs.rs/bb8/0.8.4/bb8/struct.State.html did we miss something for having the new `Statistic` struct being added in the documentation?

**Comment by djc** [comment]
Ahh, we probably need to re-export `Statistics` in `lib.rs`. Do you want to send a PR to do that, including another commit to bump the version to 0.8.5? (You can run `cargo doc --open --no-deps` to check the generated docs before submitting the PR.)

**Comment by pfreixes** [comment]
Yeps, give me some seconds

**Comment by pfreixes** [comment]
Here we go https://github.com/djc/bb8/pull/209

**Pull request merged**